kong/db/declarative/import.lua

```diff
@@ -185,10 +243,11 @@ local function load_into_cache(entities, meta, hash)
   t:db_drop(false)
 
   local phase = get_phase()
+  yield(false, phase)   -- XXX
```

Comment:
Is this needed?

Reply:
@bungle ha, I found on EE side, we have it here: https://github.com/Kong/kong-ee/blob/fb705db4e1c9292500d3a7803428ac5b7102d821/kong/db/declarative/import.lua#L206, but on CE side, there does have this line.

Not sure.

Reply:
@outsinre, if it makes the code more consistent then we should do it. Thanks for making code more consistent. Makrking this as resolved.